### ISSUE TITLE
Kirki 4 breaks 'active_callback' feature with || (OR) logic

### ISSUE THREAD

**Issue by artkrsk** [presscodes/kirki#2426]
### Issue description:
In some of my panels I use "OR" conditional logic. In Kirki 3.x the following condition worked perfectly:

```php
array(
  array(
    'setting' => 'menu_style',
    'value'   => 'fullscreen',
  ),
  array(
    array(
      'setting' => 'fullscreen_sidebar_style',
      'value'   => 'slider',
    ),
    array(
      'setting' => 'fullscreen_sidebar_style',
      'value'   => 'content',
    ),
  ),
)
```

However in the newest Kirki 4.0.3 instead of working condition I'm getting a frontend error in Customizer:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'get')
    at Object.checkCondition (control.js:160)
    at Object.checkCondition (control.js:127)
    at Object.showKirkiControl (control.js:94)
    at control.js:17
    at Function.dr (load-scripts.php?c=1&load[chunk_0]=jquery-core,jquery-migrate,regenerator-runtime,wp-polyfill,wp-hooks,underscore,customize-base,wp-dom-ready&ver=5.8.2:18)
    at Object.init (control.js:8)
    at HTMLDocument.<anonymous> (control.js:273)
    at e (load-scripts.php?c=1&load[chunk_0]=jquery-core,jquery-migrate,regenerator-runtime,wp-polyfill,wp-hooks,underscore,customize-base,wp-dom-ready&ver=5.8.2:2)
    at t (load-scripts.php?c=1&load[chunk_0]=jquery-core,jquery-migrate,regenerator-runtime,wp-polyfill,wp-hooks,underscore,customize-base,wp-dom-ready&ver=5.8.2:2)
```

### Version used:
4.0.3

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))

```php
$fullscreen_menu_has_sidebar = array(
  array(
    'setting' => 'menu_style',
    'value'   => 'fullscreen',
  ),
  array( // <-- OR condition
    array(
      'setting' => 'fullscreen_sidebar_style',
      'value'   => 'slider',
    ),
    array(
      'setting' => 'fullscreen_sidebar_style',
      'value'   => 'content',
    ),
  ),
);

Kirki::add_field(
  'arts',
  array(
    'type'            => 'radio-buttonset',
    'settings'        => 'fullscreen_sidebar_style',
    'label'           => esc_html__( 'Sidebar Style', 'kinsey' ),
    'section'         => 'fullscreen_sidebar',
    'default'         => 'slider',
    'priority'        => $priority++,
    'choices'         => array(
      ''       => esc_html__( 'Disabled', 'kinsey' ),
      'slider' => esc_html__( 'Slider', 'kinsey' ),
    ),
    'active_callback' => array(
      array(
        'setting' => 'menu_style',
        'value'   => 'fullscreen',
      ),
    ),
  )
);

Kirki::add_field(
  'arts',
  array(
    'type'            => 'select',
    'settings'        => 'fullscreen_sidebar_position',
    'description'     => esc_html__( 'Position', 'kinsey' ),
    'section'         => 'fullscreen_sidebar',
    'default'         => 'left',
    'priority'        => $priority++,
    'choices'         => array(
      'left'  => esc_html__( 'Left Side', 'kinsey' ),
      'right' => esc_html__( 'Right Side', 'kinsey' ),
    ),
    'transport'       => 'postMessage',
    'active_callback' => $fullscreen_menu_has_sidebar, //  <-- here is the error
  )
);
```

Due to this frontend error neither of my panels conditions (even the simple ones without "OR" logic) working. This leads to complete mess with panels in Customizer when every panel is revealed.

I've tried to fix this on my own but unfortunately no luck. I would appreciate if you give this issue a high priority. While it doesn't completely break Customizer, it makes it totally unusable in some places.

**Comment by contactjavas** [comment]
Hi @artkrsk we'll check this asap.
Thanks for posting the codes & the error messages.
Will let you know the update here.

**Comment by contactjavas** [comment]
Hi @artkrsk , we're able to replicate the issue, and it has been fixed in our new `v4.0.6` build. 
Here is the download link: [v4.0.6](https://github.com/kirki-framework/kirki/files/7803040/kirki.zip)
Please let us know if it fixes your issue.

Thank you for your awesome report!

**Comment by artkrsk** [comment]
That helped. Thank you!
